### PR TITLE
ServiceBus : Swagger Split and Added PrivateEndPointConnection API

### PR DESCRIPTION
**Pull request by v-Ajnava**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [X] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [X] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [X] Service team MUST add the "**WaitForARMFeedback**" label if the management plane API changes fall into one of the below categories. 
  - adding/removing APIs.
  - adding/removing properties.
  - adding/removing API-version. 
  - adding a new service in Azure.

<i>Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.</i>

- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by lirenhe** [comment]
@v-Ajnava, could you please fix all the errors from tool check?

**Comment by chiragg4u** [comment]
In case there is no new API been added ARM API review is not required. Removing the tag, please add with details why ARM API review is required.

**Comment by v-Ajnava** [comment]
@lirenhe all required CI have passed, can you please review the PR ?

File: specification/servicebus/resource-manager/Microsoft.ServiceBus/preview/2018-01-01-preview/servicebus-preview.json [hunk 1]
```diff
@@ -1,1437 +0,0 @@
-{
-  "swagger": "2.0",
```
Comment (majastrz) [inline specification/servicebus/resource-manager/Microsoft.ServiceBus/preview/2018-01-01-preview/servicebus-preview.json:2]: >swagger [](start = 3, length = 7)

Was the deleted file split into all the files in the same directory?

**Comment by majastrz** [comment]
>   "swagger": "2.0",

Was the deleted file split into all the files in the same directory?

---
Refers to: specification/servicebus/resource-manager/Microsoft.ServiceBus/stable/2017-04-01/servicebus.json:2 in 792f037. [](commit_id = 792f037ddde8fc1b5280affc307c0306ebd70c00, deletion_comment = True)

**Review by majastrz — APPROVED** [review]
Signed off from ARM side.

File: specification/servicebus/resource-manager/Microsoft.ServiceBus/preview/2018-01-01-preview/servicebus-preview.json (hunk 1 above)
Comment (v-Ajnava) [reply specification/servicebus/resource-manager/Microsoft.ServiceBus/preview/2018-01-01-preview/servicebus-preview.json:2]: > > "swagger": "2.0",
> 
> Was the deleted file split into all the files in the same directory?
> 
> Refers to: specification/servicebus/resource-manager/Microsoft.ServiceBus/stable/2017-04-01/servicebus.json:2 in [792f037](https://github.com/Azure/azure-rest-api-specs/commit/792f037ddde8fc1b5280affc307c0306ebd70c00). [](commit_id = [792f037](https://github.com/Azure/azure-rest-api-specs/commit/792f037ddde8fc1b5280affc307c0306ebd70c00), deletion_comment = True)

yes, the deleted file is split in all the files in the same directory.

**Comment by v-Ajnava** [comment]
@lirenhe , can you please review the PR

**Review by lirenhe — APPROVED** [review]

**Pull request merged**